src/main/kotlin/org/digma/intellij/plugin/ui/list/summaries/SummaryPanelListCellRenderer.kt

```diff
@@ -11,15 +11,11 @@ import org.digma.intellij.plugin.model.rest.insights.SpanDurationChangeInsight
 import org.digma.intellij.plugin.model.rest.insights.SpanDurationsPercentile
 import org.digma.intellij.plugin.model.rest.insights.TopErrorFlowsInsight
 import org.digma.intellij.plugin.service.ErrorsActionsService
-import org.digma.intellij.plugin.service.InsightsActionsService
 import org.digma.intellij.plugin.ui.common.CopyableLabelHtml
 import org.digma.intellij.plugin.ui.common.Laf
 import org.digma.intellij.plugin.ui.common.asHtml
 import org.digma.intellij.plugin.ui.common.buildLinkTextWithGrayedAndDefaultLabelColorPart
-import org.digma.intellij.plugin.ui.list.AbstractPanelListCellRenderer
-import org.digma.intellij.plugin.ui.list.PanelsLayoutHelper
-import org.digma.intellij.plugin.ui.list.RoundedPanel
-import org.digma.intellij.plugin.ui.list.commonListItemPanel
+import org.digma.intellij.plugin.ui.list.*
```

Comment:
please modify your intellij so it won't make those *.
We rather see the exact imported classes without hiding them.
see [this post](https://intellij-support.jetbrains.com/hc/en-us/community/posts/206294659-Telling-Intellij-to-do-explicit-imports-instead-of-Java-)

Reply:
Fixed